google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/executor/AppEngineStandardRunTask.java

```diff
@@ -44,9 +47,11 @@
    * @param runnerId typically "Run" or "Debug", to indicate type of local run. To be used in
    *     metrics
    */
-  public AppEngineStandardRunTask(@NotNull RunConfiguration runConfig, @Nullable String runnerId) {
+  public AppEngineStandardRunTask(@NotNull RunConfiguration runConfig, @Nullable String runnerId,
+      boolean useDevappserver2) {
     this.runConfig = runConfig;
     this.runnerId = runnerId;
+    this.useDevappserver2 = useDevappserver2;
```

Comment:
nitpicking: 

camel case DevAppServer2

Also perhaps make it devAppServer2 instead of useDevAppServer2 so its more inline with pojo boolean naming conventions: e.g. the getter/setter will be isDevAppServer2/setDevAppServer2